include/verilated_funcs.h

```diff
@@ -233,6 +233,54 @@ static inline std::string VL_CVT_N_CSTR(const char* lhsp) VL_PURE {
     return lhsp ? std::string{lhsp} : ""s;
 }
 
+template <std::size_t N_Depth>
+static inline VlQueue<std::string> VL_CVT_UNPACK_TO_Q(const VlUnpacked<std::string, N_Depth>& q) VL_PURE {
+    VlQueue<std::string> ret;
+    for (size_t i = 0; i < N_Depth; ++i)
+        ret.push_back(q[i]);
+    return ret;
+}
```

Comment:
Perhaps could template the std::string argument also and use it for all the functions?

Reply:
Oops! Was using VL_UN/PACK_II as a guide and just forgot I could just template the queue data type as well, should be fixed now.